content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,619 @@
+### Check Configuration
+
+Once a contact has been defined, we can now apply which contact(s) to use in a
+check configuration. The use case for having a contact defined on a check could 
+be that the check is owned by a particular team or group. For instance:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  }
+}
+{{< /highlight >}}
+
+### Client Configuration
+
+A client configuration can include a contact to use. This is helpful for defining ownership of the client itself.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+In this section we will be showcasing how contact routing behaves depending on the various possible configurations.
+
+## Global Configuration
+
+For the examples we'll be working with we want to define what our global handlers will be.
+
+In the following section we'll be going over different scenarios and showcasing how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and PagerDuty as our example integrations/handlers.
+
+Below is the default configuration for the integrations we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "slack": {
+    "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+    "username": "sensu",
+    "channel": "#alerts",
+    "timeout": 10
+  },
+  "email": {
+    "smtp": {
+      "address": "smtp.example.com",
+      "port": 587,
+      "openssl_verify_mode": "none",
+      "enable_starttls_auto": true,
+      "authentication": "plain",
+      "user_name": "postmaster@example.com",
+      "password": "SECRET"
+    },
+    "to": "default@example.com",
+    "from": "noreply@example.com",
+    "timeout": 10
+  },
+  "pagerduty": {
+    "service_key": "someservicekey ",
+    "timeout": 10
+  }
+}
+{{< /highlight >}}
+
+## Single Handler
+
+### Single Handler with a Single Matching Contact
+
+For our first configuration we have a single handler **email** and a contact **support** configured. The **support** contact has a configuration override for **email** that will change the default _to_ email address from "default@example.com" to "support@sensuapp.com".
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Because the contact configuration matches the handler being used, any event generated for this check will use this contact's configuration.
```

Comment:
```suggestion
Because the `support` contact's configuration provides overrides matching the handler being used, any event generated for this check will use this contact's configuration.
```